botfront/server/migrations.js

```diff
@@ -253,6 +253,52 @@ Migrations.add({
         });
     },
 });
+Migrations.add({
+    version: 10,
+    up: async () => {
+        const checkIsTypeButtons = (content) => {
+            // the only type defining key in the response content should be buttons
+            const key = 'buttons';
+            const included = ['image', 'buttons', 'elements', 'custom', 'attachment', 'quick_replies']
+                .filter(k => Object.keys(content).includes(k));
+            if (key) return included.length === 1 && included[0] === key;
```

Comment:
key will always be true since it has been define just above as 'buttons', but my linter is not picking it up some am I missing something here ?

Reply:
Ok, I have tried it it's fine, the migrations does not mess up the carousels, because the buttons key in carousels is nested :  
```yaml
template_type: genericelements:  - title: teee    subtitle: tt    image_url: ''    buttons:      - title: ww        type: postback        payload: /chitchat.bye```

Reply:
yes you are right, I copy-pasted this code from bot response utils (didn't want it to be dependant on code that might change) and I forgot to remove that part

Reply:
I have removed the un-needed check